pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "rapids-pre-commit-hooks"
+version = "0.0.0"
+authors = [
+    { name = "RAPIDS Development Team", email = "pypi@rapids.ai" }
```

Comment:
Is this a real email that we would respond to? I would rather just leave one out unless it really is a maintained address.

Reply:
I assume it is real, because it was listed in another package. If it is not real, we should fix that upstream. cc: @ajschmidt8 Can you confirm?

Reply:
@raydouglass confirmed that this email would be received and could be set up to forward to whatever list of people needed.